index.js

```diff
@@ -5,6 +5,8 @@ const downstreamElectron = require('./api/index');
 // TESTING PRODUCTION
 // const downstreamElectron = require('./dist/index');
 
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```

Comment:
I guess this is only needed if you want to use servers with self signed certificate, but there should be some explanation here and I would rather move it to `readme.md`

Reply:
Your are right, a comment would be welcome